src/env.h

```diff
@@ -64,6 +56,18 @@ namespace node {
 #define NODE_PUSH_VAL_TO_ARRAY_MAX 8
 #endif
 
+// PER_ISOLATE_* macros: we're going slightly crazy with macros here but the
```

Comment:
I took out that sentence entirely and it doesn't seem we lose anything, see what you think.